terraform/main.tf

```diff
@@ -14,7 +14,7 @@
 
 locals {
   project_id = "guardian-i-42c69c"
-  name       = "test-change"
+  name       = "test-change-test"
```

Comment:
Why this change?

Reply:
this is a "unit" test to force workflows to run the guardian repo.